bus-mapping/src/rpc.rs

```diff
@@ -199,4 +218,48 @@ mod rpc_tests {
         );
         assert!(!trace_by_hash[0].struct_logs.is_empty());
     }
+
+    // The test is ignored as the values used depend on the Geth instance used
+    // each time you run the tests. And we can't assume that everyone will
+    // have a Geth client synced with mainnet to have unified "test-vectors".
+    #[ignore]
+    #[tokio::test]
+    async fn test_get_proof() {
+        let transport = Http::new(Url::parse("http://localhost:8545").unwrap());
+        let prov = GethClient::new(transport);
+
+        // taken from RPC docs https://eips.ethereum.org/EIPS/eip-1186
+        let address =
+            Address::from_str("0x7F0d15C7FAae65896648C8273B6d7E43f58Fa842")
+                .unwrap();
+        let keys = vec![Word::from_str("0x56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421").unwrap()];
+        let proof = prov
+            .get_proof(address, keys, BlockNumber::Latest)
+            .await
+            .unwrap();
+        const EXAMPLE_PROOF: &str = r#"{
+            "accountProof": [
+                "0xf90211a...0701bc80",
+                "0xf90211a...0d832380",
+                "0xf90211a...5fb20c80",
+                "0xf90211a...0675b80",
+                "0xf90151a0...ca08080"
```

Comment:
Unsure why you don't have the full hash values here. 

How can the assertion suceed if we recieve the full hashes from the RPC call but here we declare them on this way?

Reply:
It is taken from https://eips.ethereum.org/EIPS/eip-1186, need to fix it with some real data after #202 merged.

Reply:
I'd prefer to have a valid test vector before merging this PR.  I understand that maybe it's better to wait for #202 so that we can have a reproducible test :)

Reply:
Sure thing:)